Control verbosity of debug output via build config (#58)
NOAA-OWP/topmodel: docs/BMI_ADAPTION.md

- 	All output files (`topmod.out` and `hyd.out`), if opened, are closed during `bmi.finalize()` 

### Console Prints

> `printf()` statements are now handled by `TOPMODEL_DEBUG`, though this now is set via the CMake variable `DEBUG_VERBOSITY` when generating the CMake build directory. Note: Macro uses greater than or equal to logic `>=`; i.e. a setting of `2` will include `0`, `1`, & `2`. Verbosity levels are, 

Reviewer: I would phrase this without reference to history

```suggestion
`printf()` statements are controlled by `TOPMODEL_DEBUG`, set via the CMake variable `DEBUG_VERBOSITY` when generating the CMake build directory. Note: Macro uses greater than or equal to logic `>=`; i.e. a setting of `2` will include `0`, `1`, & `2`. Verbosity levels are, 
```
